packages/icon-builder/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Isn't this a separate package technically? (And not like the files ever going to change). I can have the script reach outside it's walls for it though if you prefer.